### PR TITLE
Align pagination buttons with listings column

### PR DESCRIPTION
**Pull request by anders-schneider**
## Issue

- Addresses #313 

## Description

This change makes it so that the buttons in the bottom pagination bar of the "view all listings" don't extend to the horizontal extrema and instead are aligned with the listings column.

It also removes the border from the pagination bar on the "view all listings" page.

Before:
image

After:
image

This change does not affect the prev/next buttons on the mobile view (where there is no issue with the bottom pagination bar being too wide). In the mobile view, all it does it remove the weird rounded bottom corners:

Before:
image

After:
image

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/address technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

- [X] Desktop View
- [X] Mobile View
There are six instances of `AgPagination` between the public and partner sites. I looked at each in desktop and mobile views.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have reviewed the changes in a desktop view
- [X] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes
- [ ] I have run `yarn generate:client` if I made backend changes

### CODE REVIEW

**Comment by avaleske** [comment]
hey @anders-schneider we can chat more tomorrow but I like how it looks with your changes!

To your questions, after skimming:
- Do media queries work for this?
- Are we able to move the styles to a class and add that class to the `dataPagerClassName` array, instead of adding the wrapper div?

**Comment by anders-schneider** [comment]
Thanks for taking a quick look at this a few days ago @avaleske! It's now in ready-to-actually-be-reviewed shape :)

A few notes:
- Your suggestion to use media queries was helpful! I'm doing that through the tailwind rules/styles
- I ended up keeping the structure with a new div inside the outer div. Without that (and instead taking the suggestion of just applying styles on the existing outer div), I couldn't figure out a way to not have weird space on either side of the pagination bar:

image

And two questions:
- Is `pagination-bar` an okay name for the new class I've created? It doesn't seem to match the other `data-pager` classes, but `data-pager` is less intuitive to me
- Is `ag_grid.scss` the right home for this new class? I just put it there because the styles for similar/nearby html elements were in there.

**Comment by avaleske** [comment]
I think there's a regression on mobile, the paginator should look like
image, with the buttons spaced apart.

I think it's an easy fix. testing something one sec.

File: ui-components/src/global/vendor/AgPagination.tsx [hunk 1]
```diff
@@ -45,33 +47,15 @@ const AgPagination = ({
     dataPagerClassName.push("sticky")
```
Comment (avaleske) [inline ui-components/src/global/vendor/AgPagination.tsx:47]: I don't think the `flex flex-col md:flex-row` classes on line 43 are actually doing anything, so maybe we should remove them?

File: ui-components/src/global/vendor/AgPagination.tsx [hunk 2]
```diff
@@ -45,33 +47,15 @@ const AgPagination = ({
...
   }
 
+  const paginationBarClassName = ["pagination-bar"]
```
Comment (avaleske) [inline ui-components/src/global/vendor/AgPagination.tsx:50]: nit: I might name this `data-pager` and rename the current `data-pager` to `data-pager-container` or something similar.

File: ui-components/src/global/vendor/AgPagination.tsx (hunk 1 above)
Comment (anders-schneider) [reply ui-components/src/global/vendor/AgPagination.tsx]: Done!

File: ui-components/src/global/vendor/AgPagination.tsx (hunk 2 above)
Comment (anders-schneider) [reply ui-components/src/global/vendor/AgPagination.tsx]: Oh nice, I like that - updated!

**Comment by anders-schneider** [comment]
Thanks so much @avaleske for your help on chat on this!! I think I've got this in shape for another pass whenever you have time.

File: ui-components/src/global/vendor/ag_grid.scss [hunk 3]
```diff
@@ -110,15 +110,18 @@
...
 
-.data-pager {
+.data-pager-container {
   @apply p-4;
   @apply flex;
   @apply items-center;
+  @apply bg-gray-200;
+}
+
+.include-border {
```
Comment (avaleske) [inline ui-components/src/global/vendor/ag_grid.scss:120]: very optional thought:
we could nest this inside `.data-pager-container` like
```
.data-pager-container {
  &.include-border {
  ...
  }
}
```
so it generates css like
```
.data-pager-container { ... }
.data-pager-container.include-border {
   // only apply these rules to elements that have both the data-pager-container and include-border classes
}
```

that only real benefit is it makes it clear that the use case for `include-border` is in the context of `data-pager-container`

**Review by avaleske — APPROVED** [review]
Thank you!

**Pull request merged**